students/colephalen/lesson01/assignment/inventory_management/electricAppliancesClass.py

```diff
@@ -0,0 +1,22 @@
+# Electric appliances class
+from inventoryClass import inventory
+
+class electricAppliances(inventory):
+
+    def __init__(self, productCode, description, marketPrice, rentalPrice, brand, voltage):
+        inventory.__init__(self,productCode,description,marketPrice,rentalPrice) # Creates common instance variables from the parent class
+
+
+        self.brand = brand
+        self.voltage = voltage
+
+    def returnAsDictionary(self):
```

Comment:
A lot of this code is redundant since it exists in the Inventory class. You could do something like this:

outputDict = super().__init__()
outputDict['brand'] = self.brand
outputDict['voltage'] = self.voltage
return outputDict